packages/libcheck.rb

```diff
@@ -3,31 +3,22 @@
 class Libcheck < Package
   description 'A unit testing framework for C'
   homepage 'https://github.com/libcheck/check'
-  version '0.11.0-0'
-  source_url 'https://github.com/libcheck/check/releases/download/0.11.0/check-0.11.0.tar.gz'
-  source_sha256 '24f7a48aae6b74755bcbe964ce8bc7240f6ced2141f8d9cf480bc3b3de0d5616'
+  version '0.13.0'
+  source_url 'https://github.com/libcheck/check/archive/0.13.0.tar.gz'
```

Comment:
Why the switch from the releases?

Reply:
I just grabbed the link from the list of updates. I can change it.

Reply:
Yeah, that is the problem with automated checks.